test/OpenTelemetry.Instrumentation.AspNetCore.Tests/OpenTelemetry.Instrumentation.AspNetCore.Tests.csproj

```diff
@@ -40,9 +40,9 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="2.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
     <ProjectReference Include="..\TestApp.AspNetCore.2.1\TestApp.AspNetCore.2.1.csproj" />
-  </ItemGroup>
 
-  <ItemGroup>
+
     <Reference Include="System.Net.Http" />
```

Comment:
Looks like this is now in the `'$(TargetFramework)' == 'netcoreapp2.1'` ItemGroup. I'm guess this was unintentional.

Reply:
It is. Netcoreapp "package"  already includes System.Net.Http inside it. It's why i moved from it and added only for netcoreapp2.1.